pipelines/templates/compilemsbuild.yml

```diff
@@ -34,6 +34,42 @@ steps:
     msbuildArguments: '/p:BuildProjectReferences=false'
   displayName: 'Build Player WindowsStandalone32'
 
+- task: MSBuild@1
+  inputs:
+    solution: '$(Build.SourcesDirectory)\MSBuild\Projects\MixedRealityToolkit.sln'
+    msbuildArchitecture: 'x64'
+    platform: 'WSA'
+    configuration: 'InEditor'
+    createLogFile: true
+    restoreNugetPackages: false
+    msbuildVersion: '15.0'
+    msbuildArguments: '/p:BuildProjectReferences=false'
+  displayName: 'Build InEditor WSA'
+
+- task: MSBuild@1
+  inputs:
+    solution: '$(Build.SourcesDirectory)\MSBuild\Projects\MixedRealityToolkit.sln'
+    msbuildArchitecture: 'x64'
+    platform: 'Android'

```

Comment:
Do we know if the build machines have Android and iOS support installed?

Reply:
Good point. They do appear to have the Android NDK, but I can't see anything about iOS in any of the agent capabilities.